packages/component-library/styles/__test/tester.jsx

```diff
@@ -0,0 +1,4 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
```

Comment:
I mean... you could just add `: JSX.Element` in L4 😂

Reply:
Actually, it's a `JSX` file, so I don't need this at all haha